webapp-django/crashstats/crashstats/static/crashstats/css/report_index.css

```diff
@@ -2,3 +2,11 @@
 #metadata th {
     width: 15%;
 }
+
+.humanized {
+    padding-left: 3px;
+    color: rgb(111, 111, 111);
+    font-size: 11px;
```

Comment:
I think it would be better to use an ``em`` value here instead of pixels. How about ``0.9em``?